site/docs/install-windows.md

```diff
@@ -95,6 +95,7 @@ We recommend installing:
     ```
     pacman -S zip unzip patch diffutils git
     ```
+    Make sure to add `MSYS2_INSTALL_PATH/usr/bin` to your `PATH` Environmentvariable in case you want to use CMD or Powershell. 
```

Comment:
"Optional: If you want to use these tools from CMD or Powershell, add `MSYS2_INSTALL_PATH/usr/bin` to your `PATH` Environment Variable."

Bazel already has access to these binaries if you set `BAZEL_SH` to `C:\msys64\usr\bin\bash.exe`, so let's mark this step as optional.

@meteorcloudy please verify :)

Reply:
@jin is absolutely correct. MSYS is no longer a mandatory requirement for Bazel on Windows. TensorFlow needs it because they use a lot unix tools in their build. But that might not be true for other projects. So if we want to add this, it should be optional.

Reply:
And, of course, the comment should say that it is less optional if you are a TensorFlow user.